assignment-01/queries.sql

```diff
@@ -1,445 +1,447 @@
+where
+    age between 28 and 62 and salary > 31250;
 
-/* 14 */
-SELECT 
+-- 14
+-- Select all columns for everyone whose age is not more than 48 and salary not less than 21520
+select 
     *
-FROM
+from
     employee
-WHERE
-    age <= 48 AND salary >= 21520;
+where
+    age <= 48 and salary >= 21520;
 
-/* 15 */
-SELECT 
+-- 15
+-- Select firstname and age of everyone whose firstname starts with "John" and salary in the range 25000 and 35000
+select 
     first_name, age
-FROM
+from
     employee
-WHERE
-    first_name LIKE 'John%'
-        AND salary BETWEEN 25000 AND 35000;
+where
+    first_name like 'John%'
+        and salary between 25000 and 35000;
 
-/* 16 */
-SELECT 
+-- 16
+-- Select all columns for everyone by their ages in descending order
+select 
     *
-FROM
+from
     employee
-ORDER BY age DESC;
+order by age desc;
 
-/* 17 */
-SELECT 
+-- 17
+-- Select all columns for everyone by their ages in ascending order
+select 
     *
-FROM
+from
     employee
-ORDER BY age ASC;
+order by age asc;
 
-/* 18 */
-SELECT 
+-- 18
+-- Select all columns for everyone by their salaries in descending order
+select 
     *
-FROM
+from
     employee
-ORDER BY salary DESC;
+order by salary desc;
 
-/* 19 */
-SELECT 
+-- 19
+--  Select all columns for everyone by their salaries in ascending order.
+select 
     *
-FROM
+from
     employee
-ORDER BY salary ASC;
+order by salary asc;
 
-/* 20 */
-SELECT 
+-- 20
+-- Select all columns for everyone by their salaries in ascending order whose age not less than 17
+select 
     *
-FROM
+from
     employee
-WHERE
+where
     age >= 17
-ORDER BY salary ASC;
+order by salary asc;
 
-/* 21 */
-SELECT 
+-- 21
+-- Select all columns for everyone by their salaries in descending order whose age not more than 34.
+select 
     *
-FROM
+from
     employee
-WHERE
+where
     age <= 34
-ORDER BY salary DESC;
+order by salary desc;
 
-/* 22 */
-SELECT 
+-- 22
+-- Select all columns for everyone by their length of firstname in ascending order.
+select 
     *
-FROM
+from
     employee
-ORDER BY LENGTH(first_name) ASC;
+order by length(first_name) asc;
 
-/* 23 */
-SELECT 
-    COUNT(*)
-FROM
+-- 23
+-- Select the number of employees whose age is above 45
+select 
+    count(*)
+from
     employee
-WHERE
+where
     age > 45;
 
-/* 24 */
-DELIMITER $$
-create function add_five (num int)		/* Creating a function for adding five */
-returns int
-deterministic
-begin
-declare result int;
-set result = num + 5;
-return result;
-end $$
-DELIMITER ;
-
-DELIMITER $$
-create function subtract_250 (num int)		/* Creating a function for subtracting 250 */
-returns int
-deterministic
-begin
-declare result int;
-set result = num + 5;
-return result;
-end $$
-DELIMITER ;
-
-SELECT 
+-- 24
+-- Show the results by adding 5 to ages and removing 250 from salaries of all employees
+select 
     first_name,
     last_name,
     title,
-    ADD_FIVE(age) AS age,
-    SUBTRACT_250(salary) AS salary
-FROM
+    age + 5 as age,
```

Comment:
Its best to wrap expressions in paranthesis

Reply:
Wrapped expressions with paranthesis